### PR TITLE
WIP - sets up staging for the member management application

### PR DESCRIPTION
**Pull request by Tuuleh**
@osahyoun or @eyko I need a bit of help with this. Requests to log in through the staging application are failing: 

```
Fetch API cannot load https://action-staging.sumofus.org/api/stateless/auth/password. Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://members-dashboard-staging.us-west-2.elasticbeanstalk.com' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
actions.js:71 error: TypeError: Failed to fetch(…)
```

Am I missing some configuration for the headers?

### CODE REVIEW

File: Dockerrun.aws.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "Authentication": {
   },
   "Image": {
-    "Name": "osahyoun/members_dashboard:latest"
+    "Name": "soutech/members_dashboard:latest"
   },
   "Ports": [
     {
```
Comment (osahyoun) [inline Dockerrun.aws.json:9]: I should have removed this file from the repos - could you add it to gitignore?

**Comment by osahyoun** [comment]
@Tuuleh this is looking good. To squash that bug, you just need to get a certificate behind staging (with a custom domain) for CORS to work. FYI - we're using [AWS certificate manager](https://us-west-2.console.aws.amazon.com/acm/home?region=us-west-2)

File: Dockerrun.aws.json (hunk 1 above)
Comment (osahyoun) [reply Dockerrun.aws.json:9]: or do we need it? I can't see why.

Comment (Tuuleh) [reply Dockerrun.aws.json:9]: We don't really need it, no. I can remove it :)

**Comment by Tuuleh** [comment]
@osahyoun I've created a new cert for `members-staging.sumofus.org`, and I've also added that to the load balancer the staging app uses. It just gets redirected to `sumofus.org` now. I need a bit of help with the configuration and I think it's best if I consult with you tomorrow morning.

**Pull request closed without merging**